disk-mapper/internal/setup/setup.go

```diff
@@ -74,9 +75,8 @@ func New(log *logger.Logger, csp string, diskPath string, fs afero.Afero,
 // PrepareExistingDisk requests and waits for a decryption key to remap the encrypted state disk.
 // Once the disk is mapped, the function taints the node as initialized by updating it's PCRs.
 func (s *Manager) PrepareExistingDisk(recover RecoveryDoer) error {
-	s.log.Infof("Preparing existing state disk")
 	uuid := s.mapper.DiskUUID()
-
+	s.log.Infof("Preparing existing state disk (UUID: %s)", uuid)
```

Comment:
```suggestion
	s.log.With(zap.String("uuid", uuid)).Infof("Preparing existing state disk")
```
Use structured logging to add context to messages